dtfs-central-api/src/v1/controllers/utilisation-report-service/get-fee-record.controller/helpers.test.ts

```diff
@@ -32,35 +32,35 @@ describe('get-fee-record-details.controller helpers', () => {
         start: { month: 1, year: 2024 },
         end: { month: 1, year: 2024 },
       };
-      const feeRecord = FeeRecordEntityMockBuilder.forReport(utilisationReport).build();
-      feeRecord.report.reportPeriod = reportPeriod;
+      const feeRecordEntity = FeeRecordEntityMockBuilder.forReport(utilisationReport).build();
+      feeRecordEntity.report.reportPeriod = reportPeriod;
```

Comment:
This is a slight anti pattern to our builder pattern, it would be more prudent to use the `.withReportPeriod` method when building the utilisation report rather than editing it like this. 

I was going to say this wasn't a blocker but it is also actively changing an object used across multiple tests without resetting it which doesn't cause any harm in these specific tests but is something we should always avoid. So should either set this on the outer report when creating it, or create a new report in this test for this test rather than editing the outside one.